Signal/src/call/CallAudioService.swift

```diff
@@ -249,12 +248,6 @@ struct AudioSource: Hashable {
         } catch {
             owsFail("\(TAG) failed setting audio source with error: \(error) isSpeakerPhoneEnabled: \(call.isSpeakerphoneEnabled)")
         }
-
-        if call.state == .connected, !call.isOnHold {
-            audioSession.isRTCAudioEnabled = true
```

Comment:
This was sometimes** occurring before the CallKit provider had activated our audio session. This would result in the recipient not transmitting or hearing call audio.

**The reproduction for this was for the recipient to toggle speakerphone as soon as the call connected.